### ISSUE TITLE
Check and fix documentation about running pnpm and webpack in a separate thread

### ISSUE THREAD

**Issue by pleku** [vaadin/flow-and-components-documentation#1112]
#### Acceptance Criteria
- [x] This doesn't need special highlighting as it is documented on the static html page itself, but we should revise the documentation anyway to see if it still makes sense and maybe add a disclaimer about this change in 16. In the parts where running pnpm & webpack is mentioned, it can be added that those are started in parallel.

**Comment by denis-anisimov** [comment]
We don't have any dev mode related section/tutorial at all at the moment.
There is a dedicated tutorial for production mode but no any specific mentions of dev mode.
So there is noting to update.
Looks like we need some section for dev mode where a temp page will be mentioned but this is another task.
